### ISSUE TITLE
Installation on autopi

### ISSUE THREAD

**Issue by rstms68** [tarfu/gv70-tools#1]
Hi, 

Can you add instructions how to install it on autopi? 

I have a 22 KonaEV and I cant get data from any other module than BMS.

**Comment by tarfu** [comment]
Yeah, after I tested it a bit more on my car and it works better I will add some Documentation.

Right now there is not working as I want it to work.

**Comment by tarfu** [comment]
Ok, it works OK now.
I will think about how I want to publish the container so that it works ok, as just updating latest doesn't seem to work. It doesn't check if the latest label changed and won't get the change.
